website/content/docs/upgrading/index.mdx

```diff
@@ -41,9 +46,40 @@ considerations:
   is designed for you to know when and how to safely upgrade the server.
 
 Please see our [compatibility promise](/docs/upgrading/compatibility)
-for details on how to determine compatbility. This page will focus on
+for details on how to determine compatibility. This page will focus on
 how to upgrade components in both compatible and incompatible scenarios.
 
+### Server Upgrade Command
+
+As of 0.2.1, Waypoint provides a command for users to automatically upgrade
+Waypoint server in place. For each platform, Waypoint attempts to
+update the server image in place to latest by default, or to the server image
+requested from the passed in flag to the upgrade command.
+
+~> **Note:** Due to where Waypoint server stored its database in Nomad at `/alloc`,
+the automated upgrade command cannot migrate the old servers database to a new one.
+This is only an issue if running Waypoint server 0.2.0 and older. Please follow
+the manual upgrade guide in this case.
+
+When the upgrade command is executed, the following takes place:
+
+- Takes a snapshot of the current server database and save snapshot locally. The
+  snapshot file can be saved elsewhere if a new name is passed in through using
+  `-snapshot-name`, or optionally skipped all together
+- Run the upgrade for the requested platform:
+  - `docker` - This platform will force pull the image requested to update the
+    container in docker.
+  - `kubernetes` - This platform will use `kubectl` to set the image requested on
```

Comment:
we'll need to remove this reference this if we are able to extricate the `kubectl` call from the statefulset update